app/javascript/mastodon/features/ui/components/column_loading.js

```diff
@@ -3,17 +3,25 @@ import PropTypes from 'prop-types';
 
 import Column from '../../../components/column';
 import ColumnHeader from '../../../components/column_header';
+import ImmutablePureComponent from 'react-immutable-pure-component';
```

Comment:
I tried `React.PureComponent` but for some reason that resulted in more wasted renders than `ImmuablePureComponent`.